### DISCUSSION TITLE
Problems Installing the BCI package into Project.

### DISCUSSION THREAD

**Discussion by emmamclea** [IBAHCM/RPiR#472] [Archive] [answered]
Hi, 
I recently had a problem with uploading my BCI package (from practical 4) into my Diversity project. When I used *devtools::install_github("IBAHCM/emmamcleaBCI")*, as it states in the project guidelines, I get this error:

Error: Failed to install 'emmamcleaBCI' from GitHub:
HTTP error 404.
Not Found

Did you spell the repo owner (IBAHCM) and repo name (emmamcleaBCI) correctly?

After talking with Richard, it was found that there was a problem with my Github configuration in RStudio. To solve this I had to go through the instructions again, and re-adding a PAT to Github/ Rstudio. 
These instructions can be found here for anyone else having the same problem: https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat 

All the best, 
Emma

**Comment by richardreeve** [comment, marked as answer]
Glad that worked btw!

**Reply by JumaMinya** [reply]
@richardreeve @soniamitchell 

I have a similar problem, run the above instructions for adding a PAT but still problem persist.
Would you please assist to check?
The error message is similar
![image](https://user-images.githubusercontent.com/94681599/149658401-2986f548-e7e5-4a17-8652-f710abec0806.png)

**Reply by richardreeve** [reply]
@JumaMinya - I believe you have tried to install a package called `JumaMinyaBCI` from GitHub at `JumaMinya/JumaMinyaBCI`, and not `IBAHCM/JumaMinyaBCI`. You need to check you have the right location of  in the `DESCRIPTION` file.

**Reply by JumaMinya** [reply]
@soniamitchell @richardreeve 
This was solved and the appropriate edits made but the error with permission to access the repo (IBAHCM) still perstist.
I even tried runing devtools::install_github("IBAHCM/soniamitchellBCI") instead but similar error is experirnced

**Reply by richardreeve** [reply]
What error do you actually get when you try to install Sonia's package using `devtools::install_github("IBAHCM/soniamitchellBCI")`? The only reason I can see you would have a problem with that command is if you still have a problem with your PAT.

**Reply by JumaMinya** [reply]
Thanks
